app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/ImageFileLoader.kt

```diff
@@ -82,7 +82,7 @@ class ImageFileLoader(val context: Context) : CoroutineScope{
 
 
                 if (file != null && file.exists()) {
-                    if (id != null && name != null && path != null && bucketId != null && bucketName != null) {
+                    if (name != null && path != null && bucketName != null) {
```

Comment:
What warning does this fix? Are you sure there are no side effects? Thanks!

Reply:
This fixes the warning which shows 
Condition 'id != null' is always 'true'
Condition 'bucketId != null' is always 'true'

there is no harm removing them